apps/style/dance/style.scss

```diff
@@ -0,0 +1,61 @@
+$root: '/blockly/media/dance/'; //TODO: Parameterize for asset pipeline
+
+@import 'font';
+@import 'color';
+@import 'mixins';
+@import '../VisualizationOverlay.scss';
+
+.legacy-share-view {
+  #runButton, #resetButton {
+    display:none !important;
+  }
+}
+
+/* Sprited icons. */
+.icon21 {
+  height: 21px;
+  width: 21px;
+  background-image: url("#{$root}icons.png");
+  background-size: 42px 147px;
+}
+.code {
+  background-position: -21px 0px;
+}
+.link {
+  background-position: 0px 0px;
+}
+.run {
+  background-position: 0px -21px;
+  margin-right: 3px;
+}
+.stop {
+  background-position: -21px -21px;
+  margin-right: 3px;
+}
+.dpad-btn {
+  background-position: -21px -63px;
+}
+.left-btn {
+  background-position: -21px -42px;
+  margin-right: 3px;
+}
+.right-btn {
+  background-position: -21px -126px;
+  margin-right: 3px;
+}
+.up-btn {
+  background-position: 0px -63px;
+  margin-right: 3px;
+}
+.down-btn {
+  background-position: 0px -42px;
+  margin-right: 3px;
+}
```

Comment:
Seeing this duplicated all around the codebase, worth extracting out a shared buttons.scss include?